trivia.py

```diff
@@ -359,6 +365,13 @@ def _stop(self, *args):
             self._save_game()
             self.factory.running = False
 
+    def _write_log(self, logfile, f, q):
```

Comment:
I'm really not a big fan of single letter variables.

I'm guessing 'f' is 'filename' and 'q' is question, but I abhor guessing what variables are supposed to mean and prefer more semantic names for things so there is no room for ambiguity.